_episodes/11-act-actel.md

```diff
@@ -112,14 +114,13 @@ all_stations <- bind_rows(deploy_metadata, otn_deploy_metadata)
 
 # Mutate metadata into Actel format ----
 
-# Create a station entry from the projectcode and station number.
+# Create a station entry from the glatos array and station number.
```

Comment:
did you mean glatos array here? ACT array?

Reply:
copied from Bruce's code directly here, but i assume it should be projectcode. ill fix in all the spots